index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      class="viewport"
+      content="width=device-width, initial-scale=1.0"
+    />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <nav id="navbar">
+      <a href="#projects">MyProjects</a>
+      <a href=".project-tile">Project Section</a>
+      <a href="https://github.com/OkraGrey/" id="profile-link" target="_blank"
+        >My Profile</a
+      >
+    </nav>
+    <div  id="welcome-section" >
+      <h1 >Hello We two are here</h1>
+      <p >We are developers</p>
+    </div>
+
+    <section id="projects">
+      <h1  id="project-file" class="title center">Projects Title</h1>
+      <a class="project-tile" href="https://www.google.com/"></a>
+<div class="container">
+      <div class="small">
+        <h1 class="small-heading">
+          Img Heading
+        </h1>
+        <p class="small-description">
+          Image Description
+        </p>
+        <img  class="imgs" src="https://images.pexels.com/photos/2246476/pexels-photo-2246476.jpeg?auto=compress&cs=tinysrgb&w=600" alt="">
+      </div>
+
+      <div class="small">
+        <h1 class="small-heading">
+          Img Heading
+        </h1>
+        <p class="small-description">
+          Image Description
+        </p>
+        <img  class="imgs" src="https://images.pexels.com/photos/2246476/pexels-photo-2246476.jpeg?auto=compress&cs=tinysrgb&w=600" alt="">
+      </div>
+      <div class="small">
+        <h1 class="small-heading">
+          Img Heading
+        </h1>
+        <p class="small-description">
+          Image Description
+        </p>
+        <img  class="imgs" src="https://images.pexels.com/photos/2246476/pexels-photo-2246476.jpeg?auto=compress&cs=tinysrgb&w=600" alt="">
+      </div>
+
+    </div>
+<footer>
+  <center>
+  <a class="project-tile" href="https://www.google.com/">Project1</a>
+</center>
+</footer>
+
+    </section>
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <script src="js/index.js"></script>
+  </body>
+</html>
```

Comment:
Please use correct html semantic tags and use h1 tag only for major headings. You can use h2-h6 for other smaller headings within the project.